docs/getting-started-guides/gce.md

```diff
@@ -15,46 +15,35 @@ The example below creates a Kubernetes cluster with 4 worker node Virtual Machin
 
 1. You must be running Linux or Mac OS X on your workstation.
 2. You must have the [Google Cloud SDK](https://developers.google.com/cloud/sdk/) installed.  This will get you `gcloud` and `gsutil`.
-3. Install `gcloud preview`: run `gcloud components update preview` to make sure it is.
-4. Ensure that your other `gcloud` components are up-to-date by running `gcloud components update`.
-5. If you want to build your own release, you need to have [Docker installed](https://docs.docker.com/installation/).  On Mac OS X you can use [boot2docker](http://boot2docker.io/). (see also: https://docs.docker.com/installation/mac/)
-6. Get or build a [binary release](binary_release.md) of Kubernetes.
+3. Ensure that your other `gcloud` components are up-to-date by running `gcloud components update`.
```

Comment:
We actually do require gcloud preview, unless the last gcloud push promoted MIGs.